docs/public/specifications/tests/Address-TP/address01.md

```diff
@@ -10,38 +21,122 @@ name servers must have addresses in the routable public addressing space.
 
 IANA is responsible for global coordination of the IP addressing system.
 Aside its address allocation activities, it maintains reserved address ranges
-for special uses. These ranges can be categorized into three types : 
-[Special purpose IPv4
-addresses](https://www.iana.org/assignments/iana-ipv4-special-registry/iana-ipv4-special-registry.xml),
-[Special purpose IPv6
-addresses](https://www.iana.org/assignments/iana-ipv6-special-registry/iana-ipv6-special-registry.xml)
-and [Multicast reserved
-addresses](https://www.iana.org/assignments/multicast-addresses/multicast-addresses.xml).
+for special uses. These ranges can be categorized into three types:
```

Comment:
```suggestion
for special uses. These ranges can be categorized into two types:
```

And even then, maybe the list below could be made inline. There is little point in having an itemized list with only two items if these items are short.